timestamp-impl/src/main/java/com/palantir/timestamp/AvailableTimestamps.java

```diff
@@ -30,7 +30,9 @@
     private final PersistentUpperLimit upperLimit;
 
     public AvailableTimestamps(LastReturnedTimestamp lastReturnedTimestamp, PersistentUpperLimit upperLimit) {
-        DebugLogger.logger.info("Creating AvailableTimestamps object on thread {}. This should only happen once.",
+        DebugLogger.logger.info("Creating AvailableTimestamps object on thread {}. This should only happen once."
```

Comment:
nit: I think the `This should only happen once.` here should be removed (probably hanging from copypaste)